bedrock/firefox/tests/test_views.py

```diff
@@ -641,3 +639,74 @@ def test_firefox_welcome_page1(self, render_mock):
         req.locale = "en-US"
         views.firefox_welcome_page1(req)
         render_mock.assert_called_once_with(req, "firefox/welcome/page1.html", ANY, ftl_files="firefox/welcome/page1")
+
+
+@override_settings(
+    TWILIO_ACCOUNT_SID="",
+    TWILIO_AUTH_TOKEN="",
+    TWILIO_MESSAGING_SERVICE_SID="",
+)
+class TestSMSSendToDevice(TestCase):
+    def setUp(self):
+        self.url = "/en-US" + reverse("firefox.sms-send-to-device-post")
+
+    def _do_post(self, params):
+        return self.client.post(
+            self.url,
+            data=params,
+        )
+
+    @patch("bedrock.firefox.views.TWILIO_CLIENT")
+    def test_no_data(self, mock_client):
+        resp = self._do_post({})
+        assert resp.status_code == 200
+        data = resp.json()
+        assert data["success"] is False
```

Comment:
nit: better to just use `assert not data["success"]`? Don't normally use the `is` keyword with booleans, but I guess it is more specific. Either way.

Reply:
Makes sense. This follows the pattern already in use in this file in other tests. But I agree simply `if cond:` or `if not cond:` is more Pythonic.